src/app/docs/kagent/resources/release-notes/page.mdx

```diff
@@ -0,0 +1,334 @@
+        envFrom:
+        - name: "GRAFANA_URL"
+          value: my-url.com
+        - name: "GRAFANA_API_KEY"
+          valueFrom:
+            type: Secret
+            key: "grafana"
+            valueRef: kagent-toolserver-secret
+    description: ""
+  ```
+</div>
+
+<div id="kmcp-tab">
+  kmcp example:
+  * The MCPServer includes the Grafana deployment details, with `stdio` as the transport type.
+  * The Grafana API key is loaded separately in a Secret, which is more secure.
+  ```yaml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: grafana-api-key
+  type: Opaque
+  data:
+    GRAFANA_API_KEY: my-base-64-ikey
+  ---
+  apiVersion: kagent.dev/v1alpha1
+  kind: MCPServer
+  metadata:
+    name: grafana
+  spec:
+    deployment:
+      image: "mcp/grafana:latest"
+      port: 3000
+      cmd: "/app/mcp-grafana"
+      args:
+        - "--transport"
+        - "stdio"
+      env:
+        GRAFANA_URL: my-url.com
+      secretRefs:
+        - name: grafana-api-key
+    transportType: "stdio"
+  ```
+</div>
+
+#### HTTP ToolServer moved to RemoteMCPServer
+
+ToolServer resources that used `type: streamableHttp` are now configured as RemoteMCPServer resources. For more detailed information, review the API definitions:
+* ToolServer: [toolserver_types.go](https://github.com/kagent-dev/kagent/blob/main/go/controller/api/v1alpha1/toolserver_types.go)
+* RemoteMCPServer: [remotemcpserver_types.go](https://github.com/kagent-dev/kagent/blob/main/go/controller/api/v1alpha2/remotemcpserver_types.go)
+* MCPServer: [mcpserver_types.go](https://github.com/kagent-dev/kmcp/blob/main/api/v1alpha1/mcpserver_types.go)
+
+<Tabs tabs={[
+  { id: 'old-http', label: 'Old ToolServer (v1alpha1)' },
+  { id: 'new-http', label: 'New RemoteMCPServer (v1alpha2)' }
+]} />
+
+<div id="old-http-tab">
+  Old ToolServer API:
+  ```yaml
+  apiVersion: kagent.dev/v1alpha1
+  kind: ToolServer
+  metadata:
+    name: kagent-tool-server
+  spec:
+    config:
+      type: streamableHttp
+      streamableHttp:
+        url: "http://kagent-tools.kagent:8084/mcp"
+        timeout: 30s
+        sseReadTimeout: 5m0s
+    description: "Official KAgent tool server"
+  ```
+</div>
+
+<div id="new-http-tab">
+  New RemoteMCPServer API:
+  ```yaml
+  apiVersion: kagent.dev/v1alpha2
+  kind: RemoteMCPServer
+  metadata:
+    name: kagent-tool-server
+  spec:
+    url: "http://kagent-tools.kagent:8084/mcp"
+    timeout: 30s
+    sseReadTimeout: 5m0s
+    description: "Official KAgent tool server"
+  ```
+</div>
+
+### Kubernetes Services as HTTP MCP servers
+
+Now, you can use Kubernetes Services as MCP Servers.
+
+In the following configuration file, notice the following settings:
+* `kagent.dev/mcp-service: "true"`: Optional: Configure kagent to discover the tools for this Service.
+* `appProtocol: mcp`: Required: Configure the port that the controller uses. If not set, and there is a single port, the controller uses that port for the MCP server.
+
+```yaml
```

Comment:
Can you do the same thing here with `tabs`? That seems like a pretty good way to visualize the differences